src/nvapi.cpp

```diff
@@ -134,28 +136,37 @@ extern "C" {
         return Ok(str::format(n, " ", nr, " (", error, ")"));
     }
 
+    static std::mutex initializationMutex = std::mutex{};
```

Comment:
I think we can use `auto` here. Also on the next line with `… = 0UL`.

Reply:
And done.